robot_skills/src/robot_skills/util/nav_analyzer.py

```diff
@@ -142,7 +142,7 @@ def stop_measurement(self, endpose, result):
 
         ''' Log endpose '''
         endposeitem = ET.SubElement(self.logitem, "endpose")
-        self.poseStampedToSubElement(endpose, endposeitem)
```

Comment:
It's only used in count_{reset,clear,plan}, which are themselves never used in that file so the whole lot can be removed.